lib/services/local/trust.go

```diff
@@ -121,6 +122,9 @@ func (s *CA) CompareAndSwapCertAuthority(new, existing types.CertAuthority) erro
 	_, err = s.CompareAndSwap(context.TODO(), existingItem, newItem)
 	if err != nil {
 		if trace.IsCompareFailed(err) {
+			if len(existing.GetMetadata().Labels) >= 2 {
+				log.Warn("compare-and-swap comparison failed on certificate authority with multiple labels; it may need to be re-saved to sort map keys")
```

Comment:
Any warnings or errors we log should be actionable and easy to understand by an end user, not just developers.

If a user sees this warning message, what do we expect them to do about it?

- if we don't expect them to do anything, then it's probably not a warning
- if we do expect them to do something, let's make the message more clear as to _what_ that is

Reply:
Fair point, I've changed it to now output a message like the following:

> comparison failed on certificate authority with multiple labels; if this occurs consistently, try re-saving the resource: tctl get cert_authority/user/foo.example.com --with-secrets > ca.yaml && tctl create -f ca.yaml && rm ca.yaml